### DISCUSSION TITLE
SwinUNETR input image_size problem.

### DISCUSSION THREAD

**Discussion by justinzhang528** [Project-MONAI/tutorials#1412] [Q&A]
Dear MONAI Community,

I used image_size 128x128x48 to train [SwinUNETR](https://docs.monai.io/en/stable/_modules/monai/networks/nets/swin_unetr.html#SwinUNETR.__init__) model but the error showed "input image size (128x128x48) should be divisible by stage-wise image resolution". I was confused about why my image_size didn't work?

Thanks in advanced!

**Comment by tangy5** [comment]
Hi Justin, thanks for the question,  the swin transformer for 3D requires a downsample/upsample factors of 16, can you try 64 in the third dimension?

**Reply by justinzhang528** [reply]
It works for me to set 64 in the 3rd dimension, thanks. So the resolution should be divisible by 32 ?

**Reply by tangy5** [reply]
It might be at least 64. Then 96, 128 also work.

**Reply by justinzhang528** [reply]
got it, thanks.
